LoRaEngine/test/LoRaWanNetworkServer.Test/LoRaPayloadDecoderTest.cs

```diff
@@ -16,6 +16,10 @@ public class LoRaPayloadDecoderTest
         [InlineData("DecoderValueSensor", "1234", 2)]
         [InlineData("DECODERVALUESENSOR", "1234", 1)]
         [InlineData("decodervaluesensor", "1234", 2)]
+        [InlineData("decodervaluesensor", "hello world", 2)]
+        [InlineData("decodervaluesensor", " 1 ", 2)]
+        [InlineData("decodervaluesensor", "$1", 2)]
+        [InlineData("decodervaluesensor", "100'000", 2)]
         public async Task When_Decoder_Is_DecoderValueSensor_Return_In_Value(string decoder, string payloadString, byte fport)
```

Comment:
Nice one. Could you please modify the test (including name) ABP_Unconfirmed_With_Value_Decoder_Sends_Decoded_Numeric_Payload to handle other payloads (strings, doubles, etc).

This is a e2e test without external dependencies. It checks that the data we are sending to IoT Hub is correct, after the value has been decoded. If you need help please let me know.

Reply:
Sure, I added string payloads to that test as well.